cluster_tools/cluster_tools/schedulers/slurm.py

```diff
@@ -200,6 +200,25 @@ def submit_text(cls, job, cfut_dir):
     def handle_kill(self, *args, **kwargs):
         for submit_thread in self.submit_threads:
             submit_thread.stop()
+
+        # Jobs with a "pending" state have not been submitted to the cluster yet
+        scheduled_job_ids = [
+            job_id for job_id, job_state in self.jobs.items() if job_state != "pending"
```

Comment:
the pending-check covers this case here, right?

```
                # Register the job in the jobs array, although the jobid is not known yet.
                # Otherwise it might happen that self.jobs becomes empty, but some of the jobs were
                # not even submitted yet.
                self.jobs[workerid_with_index] = "pending"
```

can you rename "pending" in both places to something else so that it cannot be confused with slurm's pending state. this caught me off guard for a moment. how about "unsubmitted" or "not_yet_submitted"? could be in all caps, too.

Reply:
Yes, that's right, Good point, I'll rename to `NOT_YET_SUBMITTED`.